cosavostra/localise-bundle/1.2/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "CosaVostra\\LocaliseBundle\\CosaVostraLocaliseBundle": ["all"]
+    },
+    "env": {
+        "LOCALISE_EXPORT_KEY": "localise_api_key_to_replace"
```

Comment:
I dont see that this is used by your bundle's configuration.

Reply:
@Nyholm Check https://github.com/mradhi/localise-bundle/blob/master/Resources/config/services.xml, I used this env variable in `CosaVostra\LocaliseBundle\Http\Client` service

Reply:
You are referencing me to your service definition. 
Your bundle dont even have configuration: https://github.com/mradhi/localise-bundle/tree/master/DependencyInjection

The issue with your current setup is that if a user is missing the environment variable, they will not get an exception until runtime.
I suggest you create some configuration to your bundle and make sure to use that env var there. If the env var is missing then, the user will get an error at build time, which is much more preferred.

Reply:
Friendly ping